layouts/partials/head/styling.html

```diff
@@ -2,6 +2,6 @@
 {{ $css := resources.Get "css/main.css" }}
 {{ $css = $css | resources.PostCSS }}
```

Comment:
Hi @135yshr, I think you meant to change this line instead of the other one 😇. Please have a 2nd look 🙏🏻 .

Reply:
Thanks for the review & bug fixs.